server/routes/api.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express')
+const router = express.Router()
+const axios = require('axios')
+const url = require(`urllib`)
+require('dotenv').config()
+
+const City = require('../model/City')
+
+const apiKey = process.env.API_KEY
+
+router.get('/city/:cityName', function (req, res) {
+  const { cityName } = req.params
+
+  url.request(
+    `http://api.openweathermap.org/data/2.5/weather?q=${cityName}&appid=${apiKey}&units=metric`,
+    function (err, cityData) {
+      //axios.get didn't work for me ????????????????????????
```

Comment:
it should work, maybe you did not take the `response.data` that is where you see all the data that is sent back to you